### ISSUE TITLE
'bool' object has no attribute 'date'

### ISSUE THREAD

**Issue by ColinRobbins** [EJEP/datapoint-python#148]
Hello,
I not sure if this is the right place to raise this, but I am seeing the follow error when use in homeassistant...
```
Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/srv/homeassistant/lib/python3.7/site-packages/homeassistant/helpers/entity.py", line 286, in async_update_ha_state
    self._async_write_ha_state()
  File "/srv/homeassistant/lib/python3.7/site-packages/homeassistant/helpers/entity.py", line 328, in _async_write_ha_state
    attr.update(self.device_state_attributes or {})
  File "/srv/homeassistant/lib/python3.7/site-packages/homeassistant/components/metoffice/sensor.py", line 172, in device_state_attributes
    attr[ATTR_LAST_UPDATE] = self.data.data.date
AttributeError: 'bool' object has no attribute 'date'
```
I’m not really sure what is going on, but clearly the self.data.data does not contain a date!

Where can I find a description of what the returned data structure should be, so I can try to find out more about what is happening?

Any ideas what the ‘bool’ object is likely to be?

Sorry for the novice question, but this is a new API to me, and just trying to help track down an error!

**Comment by EJEP** [comment]
Please can you let me know what version of datapoint you are using? Until recently (24th December 2019) homeassistant was using version 0.4.3 of datapoint. This pull request bumped the version of datapoint used by homeassistant to 0.9.5: https://github.com/home-assistant/home-assistant/pull/30185.

A number of bugs relating to this sort of error have been fixed since version 0.4.3, so upgrading may solve this issue. This issue may be a manifestation of the issue in #27.

**Comment by ColinRobbins** [comment]
I'm using Home Assistant 0.103.6, which I pulled on Jan 11.  
That places the dependency on 0.4.3, so seems that's where my issue is.
I've just checked the development branch of Home Assistant, and looks like it updated to 0.9.5 just after that.
So hopefully the issue will be resolved on the next update.
Thanks.
